app/controllers/users_controller.rb

```diff
@@ -1,29 +1,24 @@
 class UsersController < ApplicationController
-  LOCALES = [
-              ["English", "en"],
-            ]
-
-  def settings
-    load_locales
+  def edit
+    @user = User.find(params[:id])
   end
 
-  def update_settings
-    params = user_params
-    if current_user.update(params)
-      redirect_to settings_path, notice: "Settings updated"
-    else
-      load_locales
-      render :settings
+  def update
+    @user = User.find(params[:id])
+
+    if can_edit? && @user.update(admin_update_params)
+      @_message = "User updated"
     end
-  end
 
+    redirect_to edit_user_path(@user), notice: @_message
+  end
   private
 
-  def user_params
-    params.require(:user).permit(:locale, :time_zone, :timestamps_in_device_time_zone)
+  def admin_update_params
+    params.require(:user).permit(:is_active)
   end
 
-  def load_locales
-    @locales = LOCALES
+  def can_edit?
+    current_user.id != params[:id].to_i
```

Comment:
I'd check on the already loaded user rather than the params, in case we start using a different parameter (uuid?) in the future. So you can do `current_user != @user` directly.